asset/quickstart/analyze-iam-policy-longrunning-bigquery/main.go

```diff
@@ -66,6 +66,16 @@ func main() {
 		log.Fatalf("AnalyzeIamPolicyLongrunning: %v", err)
 	}
 	fmt.Print(op.Metadata())
+
+	// Wait for the longrunning operation complete.
+	resp, err := op.Wait(ctx)
+	if err != nil && !op.Done() {
+		fmt.Println("failed to fetch operation status", err)
+	} else if err != nil && op.Done() {
+		fmt.Println("operation completed with error", err)
+	} else {
+		fmt.Println("operation completed successfully", resp)
+	}
```

Comment:
Might be good to switch these to returning early to keep indentation at a minimum for the happy path:

```suggestion
	if err != nil && !op.Done(){
		fmt.Println("failed to fetch operation status", err)
		return
	}
	if err != nil && op.Done(){
		fmt.Println("operation completed with error", err)
		return
	}
	fmt.Println("operation completed successfully", resp)
```

https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow